internal/commands/result.go

```diff
@@ -399,16 +399,17 @@ func parseResultsSonar(results *wrappers.ScanResultsCollection) []wrappers.Sonar
 		for _, result := range results.Results {
 			auxIssue.Severity = severities[result.Severity]
 			auxIssue.Type = vulnerabilitySonar
-			// CxSAST result type
-			if result.Type != kicsTypeLabel {
+
+			if strings.EqualFold(strings.TrimSpace(result.Type), sastTypeLabel) {
 				auxIssue.EngineID = result.Type
 				auxIssue.RuleID = result.ID
 				auxIssue.PrimaryLocation = parseSonarPrimaryLocation(result)
 				auxIssue.SecondaryLocations = parseSonarSecondaryLocations(result)
 				auxIssue.EffortMinutes = 0
 				sonarIssues = append(sonarIssues, auxIssue)
-				// Infrastructure result type
-			} else {
+			}
+
+			if strings.EqualFold(strings.TrimSpace(result.Type), kicsTypeLabel) {
```

Comment:
Could be an else if, no need to do another comparison if the first one matches

Reply:
Don't you think it becomes a bit more readable? Having the Ifs separate makes it easier to understand, avoiding more nesting.